ui/app/components/clients/attribution.js

```diff
@@ -1,45 +1,105 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 
-// TODO: fill out below!!
 /**
  * @module Attribution
- * Attribution components are used to...
+ * Attribution components display the top 10 total client counts for namespaces or auth methods (mounts) during a billing period.
+ * If view is filtered for a single month, two graphs display and show a breakdown of new and total client counts by namespace or auth method, respectively 
  *
  * @example
  * ```js
- * <Attribution @requiredParam={requiredParam} @optionalParam={optionalParam} @param1={{param1}}>
- * Pass in export button
- * </Attribution>
+ * <Clients::Attribution
+ *    @newClientsData={{this.newClientData}}
+ *    @totalClientsData={{this.totalClientData}}
+ *    @chartLegend={{this.chartLegend}}
+ *    @isDateRange={{this.isDateRange}}
+ *    @isAllNamespaces={{this.isAllNamespaces}}
+    >
+      <button type="button"> Export attribution data </button>
+    </Clients::Attribution>
  * ```
- * @param {object} requiredParam - requiredParam is...
- * @param {string} [optionalParam] - optionalParam is...
- * @param {string} [param1=defaultValue] - param1 is...
+
+ * @param {array} newClientsData - (passed to child chart) must be an array of flattened objects
+ * @param {array} totalClientsData - (passed to child chart) must be an array of flattened objects
+ * @param {array} chartLegend - (passed to child) array of objects with key names 'key' and 'label' so data can be stacked
+ * @param {boolean} isDateRange - discerns if dataset from API is a date range or single month
```

Comment:
I like the label name. I'll keep note of this because I have the event handler wired up to handle when they click single month and this is the param I need to modify when they do.